_literate/ISL-lab-4/tutorial.jl

```diff
@@ -118,7 +112,7 @@ cm = confusion_matrix(ŷ, y)
 @show false_positive(cm)
 @show accuracy(ŷ, y)  |> r3
 @show accuracy(cm)    |> r3  # same thing
-@show precision(ŷ, y) |> r3
+@show StatisticalMeasures.precision(ŷ, y) |> r3
```

Comment:
This might be a bit confusing, because `StatisticalMeasures`, while exported by MLJ, is not something the new user is aware of. How about this workaround?

```suggestion
@show postive_predictive_value(ŷ, y) |> r3   # a.k.a. precision
```